packages/xstate-fsm/README.md

```diff
@@ -13,7 +13,29 @@
 
 This package contains a minimal, 1kb implementation of [XState](https://github.com/davidkpiano/xstate) for **finite state machines**.
 
-**Features:**
+## Features
+
+|                             | **@xstate/fsm** | [XState](https://github.com/davidkpiano/xstate) |
```

Comment:
Suggestion - May be helpful for future readers if each of these was a link to the official docs (may be able to make a separate PR for that later if it doesn't get added to this one but is a wanted addition)